frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.test.tsx

```diff
@@ -145,9 +145,14 @@ describe("Test PollingStationChoiceForm", () => {
       // Test if the polling station name is shown
       await user.type(pollingStation, "34");
 
-      // Click submit again and see that the alert appeared again
+      // Click submit and see that the alert appears
       await user.click(submitButton);
 
+      // Test if the warning message is shown correctly
+      expect(screen.getByTestId("pollingStationSelectorFeedback").textContent).toBe(
+        "Stembureau 34 (Testplek) is al twee keer ingevoerd",
+      );
```

Comment:
@jschuurk-kr Could you have a look at this test, it works in the browser, but somehow in the test the text shows `"Geen stembureau gevonden met nummer 34"`

Reply:
Wrapping it with `waitFor` works a bit better:
```
      await waitFor(() => {
        expect(screen.getByTestId("pollingStationSelectorFeedback").textContent).toBe(
          "Stembureau 34 (Testplek) is al twee keer ingevoerd",
        );
      });
```
But I find it strange that "Geen stembureau gevonden met nummer 34" would be rendered initially.

Reply:
Hmm yeah no clue why that is, like I said it does not happen when I just test in the browser. Just goes from aan het zoeken... to the expected message 🤔